src/components/List/List.test.jsx

```diff
@@ -0,0 +1,31 @@
+import { render } from '@testing-library/react';
+import List from './index';
+import { EMPTY_MESSAGE } from '../../constants';
+
+describe('List', () => {
+  it(`tasks의 length가 0이면 "${EMPTY_MESSAGE}"를 보여줍니다.`, () => {
+    const sut = render(<List tasks={[]} />);
+    const $emptyElement = sut.queryByText(EMPTY_MESSAGE);
+
+    expect($emptyElement).not.toBeNull();
+  });
+
+  it(`tasks의 length가 0이 아니면 "${EMPTY_MESSAGE}"를 보여주지 않습니다.`, () => {
+    const sut = render(<List tasks={[{ id: 1, title: 'foo' }]} />);
+    const $emptyElement = sut.queryByText(EMPTY_MESSAGE);
+
+    expect($emptyElement).toBeNull();
```

Comment:
단순히 텍스트가 있는지 없는지 검사할 때는 이렇게 하면 더 간단할 것 같아요

```suggestion
    const { container } = render(<List tasks={[{ id: 1, title: 'foo' }]} />);

    expect(container).not.toHaveTextContent(EMPTY_MESSAGE);
```

See also
  - https://github.com/testing-library/jest-dom#tohavetextcontent

Reply:
이렇게 한 이유는, 결국 화면에 보이는 건 텍스트가 아니라 요소라서 선택했습니다. 위에 선택한 내용과는 상반되지만... 
```javascript
const $emptyElement = sut.queryByText(EMPTY_MESSAGE);

expect($emptyElement).toBeNull();
```
이렇게 했을 때 `toHaveTextContent`보단 `p` 요소 내의 요소 등의 내용이 변경되었을 때 테스트의 의미전달이 더 좋지 않을까요??

Reply:
표현력의 차이인 것 같아요 ㅎㅎ